zerver/webhooks/zabbix/view.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Dict
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile
+
+import ujson
+
+ZABBIX_SUBJECT_TEMPLATE = '{hostname}'
+ZABBIX_MESSAGE_TEMPLATE = '{status} ({severity}) alert on [{hostname}]({link}).\n{trigger}\n{item}'
+
+@api_key_only_webhook_view('Zabbix')
+@has_request_variables
+def api_zabbix_webhook(request: HttpRequest, user_profile: UserProfile,
+                       payload: Dict[str, Any]=REQ(argument_type='body'),
+                       stream: str=REQ(default='alerts')) -> HttpResponse:
+
+    body = get_body_for_http_request(payload)
+    subject = get_subject_for_http_request(payload)
+
+    check_send_stream_message(user_profile, request.client, stream, subject, body)
```

Comment:
Could you please use `check_send_webhook_message` here instead? You'll find that in `zerver.lib.webhooks.common`. :)